### PR TITLE
feat: add ubuntu 24.04 packages (NR-248002)

### PR DESCRIPTION
**Pull request by gsanchezgavier**
add ubuntu 24.04 packages

### CODE REVIEW

File: .github/workflows/ci-prerelease_linux_on_demand.yml [hunk 1]
```diff
@@ -9,10 +9,13 @@ on:
...
         description: 'Repo prefix'
         required: true
+      tag:
+        description: 'tag used for the testing release'
+        required: true
 
 env:
   GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  TAG: "0.2.0"
+  TAG: ${{ github.event.inputs.tag }}
```
Comment (rubenruizdegauna) [inline .github/workflows/ci-prerelease_linux_on_demand.yml:18]: nice 🧹

File: distributions/nr-otel-collector/manifest.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ dist:
   module: github.com/newrelic/opentelemetry-collector-releases/nr-otel-collector
   name: nr-otel-collector
   description: New Relic OpenTelemetry Collector
-  version: 0.6.0
+  version: 0.7.0
```
Comment (rubenruizdegauna) [inline distributions/nr-otel-collector/manifest.yaml:5]: Is this correct? We are not releasing anything new 🤔 I would wait until something is added and I would add to those release notes that the release adds support for Ubuntu 24.04.

Wdyt?

Comment (gsanchezgavier) [reply distributions/nr-otel-collector/manifest.yaml:5]: I'm a bit afraid not to release now after some changes in the pipeline and then have to deal with some issues in the future , this is why i was planning to do a release

**Comment by gsanchezgavier** [comment]
closed in favor of #108

**Pull request closed without merging**